subsys/nrf_security/src/drivers/cracen/cracenpsa/src/key_management.c

```diff
@@ -605,8 +607,10 @@ static psa_status_t export_ecc_public_key_from_keypair(const psa_key_attributes_
 	int si_status = 0;
 	psa_algorithm_t key_alg = psa_get_key_algorithm(attributes);
 	const struct sx_pk_ecurve *sx_curve;
+#if CONFIG_PSA_NEED_NO_SI_CRYPTO_ED25519
```

Comment:
Unsure on the usage of compiler definitions here. If more no_si_crypto functions are added it could become quite difficult to understand

Reply:
Removed config